src/api/GraphQuery/types.ts

```diff
@@ -200,6 +200,10 @@ export interface GraphQuery {
   dialect: GraphQueryDialect;
   description: string;
   sharing: GraphQuerySharingMode;
+  owner?: {
```

Comment:
@mzorkany @yigitakbulut why optional?

Reply:
@francescoinfante the built in queries have no owner, so there are two options, we either null or something like that for built in queries or make the owner object optional. What do you think?

Reply:
@mzorkany @yigitakbulut make 100% sense :) (in Rest client it's good practice to write why something can be undefined as a comment)